.pre-commit-config.yaml

```diff
@@ -4,6 +4,7 @@ repos:
     rev: v3.2.0
     hooks:
       - id: check-added-large-files
+        args: ['--maxkb=10000']
```

Comment:
what is this for?

Reply:
This is because the current limit can be restrictive for notebooks. My notebook's size is currently 1.3MB. I believe the default limit is around 800KB or so. I made it 10MB for a lot of wiggle room.

Reply:
Can you add a concise but informative comment on what this flag does?

Reply:
Done!